File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TripResponseEndCoordinates.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * End (latitude, longitude) in decimal degrees.
 * @param latitude 
 * @param longitude 
 */
data class TripResponseEndCoordinates (

        @JsonProperty("latitude") val latitude: Double? = null,

        @JsonProperty("longitude") val longitude: Double? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TripResponseEndCoordinates.scala

package org.openapitools.client.model


case class TripResponseEndCoordinates (
    _latitude: Option[Double],
    _longitude: Option[Double]
)
object TripResponseEndCoordinates {
    def toStringBody(var_latitude: Object, var_longitude: Object) =
        s"""
        | {
        | "latitude":$var_latitude,"longitude":$var_longitude
        | }
        """.stripMargin
}
